packages/react-instantsearch-dom/src/widgets/Answers.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useEffect, useMemo, useContext } from 'react';
+import { instantSearchContext } from 'react-instantsearch-core';
+import { createConcurrentSafePromise } from '../lib/createConcurrentSafePromise';
+import { debounce } from '../lib/debounce';
+
+/* eslint-disable react/prop-types */
+function DefaultAnswersComponent({ isLoading, hits = [] }) {
+  return isLoading ? (
+    <p>loading...</p>
+  ) : (
+    hits && hits.length > 0 && <pre>{JSON.stringify(hits, null, 2)}</pre>
+  );
+}
+
+export default function Answers({
```

Comment:
there's no tests at all?

Reply:
Initially I wasn't sure and wanted to get some feedback on how this component is made (no connector, directly using instantSearchContext, ...). So I didn't add tests at that time. Do you see this component being released looking like this? If so, I can add tests now.

Reply:
I don't see huge blockers, as long as the hook also gets released :)